ores/scoring_context.py

```diff
@@ -233,11 +233,14 @@ def from_config(cls, config, name, section_key="scoring_contexts"):
                     scorer_models:
                         damaging: enwiki_damaging_2014
                         good-faith: enwiki_good-faith_2014
+
                     extractor: enwiki
+
```

Comment:
No concerns really but why the added space?

Reply:
The space is required by the RST spec to denote the end of a 'list' and was showing up as a warning during the sphinx-build test:
```
/ores/scoring_context.py:docstring of ores.scoring_context.ScoringContext.from_config:8:
Definition list ends without a blank line; unexpected unindent.
```